identity-core/Cargo.toml

```
@@ -24,6 +23,7 @@ serde_json = { version = "1.0", default-features = false, features = ["std"] }
 strum = { version = "0.21", features = ["derive"] }
 subtle = { version = "2.4", default-features = false }
 thiserror = { version = "1.0", default-features = false }
+time = { version = "0.3.5", features = ["std", "serde", "parsing", "formatting"] }
```

I know we actually use the default feature set, we should just be consistent to make it easier to determine that we aren't pulling in unnecessary crates or code bloat from features etc. (some of the dependencies I've added also need this change).

```suggestion
time = { version = "0.3.5", default-features = false, features = ["std", "serde", "parsing", "formatting"] }
```